tests/pr_test.go

```diff
@@ -139,15 +150,60 @@ func GetSecretsManagerKey(sm_id string, sm_region string, sm_key_id string) *str
 	return secret.(*secretsmanagerv2.ArbitrarySecret).Payload
 }
 
-// A test to pass existing resources to the SM DA
-func TestRunExistingResourcesInstances(t *testing.T) {
+func TestRunSecretsManagerFullyConfigurableUpgradeSchematic(t *testing.T) {
+	t.Parallel()
+
+	acme_letsencrypt_private_key := GetSecretsManagerKey( // pragma: allowlist secret
+		permanentResources["acme_letsencrypt_private_key_sm_id"].(string),
+		permanentResources["acme_letsencrypt_private_key_sm_region"].(string),
+		permanentResources["acme_letsencrypt_private_key_secret_id"].(string),
+	)
+
+	// Set up a schematics test
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		TarIncludePatterns: []string{
+			"*.tf",
+			fmt.Sprintf("%s/*.tf", fullyConfigurableTerraformDir),
+			fmt.Sprintf("%s/*.tf", "modules/secrets"),
+			fmt.Sprintf("%s/*.tf", "modules/fscloud"),
+		},
+		TemplateFolder:         fullyConfigurableTerraformDir,
+		ResourceGroup:          resourceGroup,
+		Prefix:                 "sm-fc-ug",
+		Tags:                   []string{"test-schematic"},
+		DeleteWorkspaceOnFail:  false,
+		WaitJobCompleteMinutes: 60,
+	})
+
+	options.TerraformVars = []testschematic.TestSchematicTerraformVar{
+		{Name: "ibmcloud_api_key", Value: options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"], DataType: "string", Secure: true},
+		{Name: "prefix", Value: options.Prefix, DataType: "string"},
+		{Name: "region", Value: validRegions[rand.Intn(len(validRegions))], DataType: "string"},
+		{Name: "existing_resource_group_name", Value: "geretain-test-secrets-manager", DataType: "string"},
+		{Name: "service_plan", Value: "trial", DataType: "string"},
+		{Name: "existing_kms_instance_crn", Value: permanentResources["hpcs_south_crn"], DataType: "string"},
+		{Name: "kms_encryption_enabled", Value: true, DataType: "bool"},
+		{Name: "cis_id", Value: permanentResources["cisInstanceId"], DataType: "string"},
+		{Name: "ca_name", Value: permanentResources["certificateAuthorityName"], DataType: "string"},
+		{Name: "dns_provider_name", Value: permanentResources["dnsProviderName"], DataType: "string"},
+		{Name: "acme_letsencrypt_private_key", Value: *acme_letsencrypt_private_key, DataType: "string"},
```

Comment:
many of these inputs are not used by the DA?? `acme_letsencrypt_private_key`, `dns_provider_name`, `ca_name`, `cis_id` etc etc. Please review